src/generic_app.ts

```diff
@@ -33,46 +33,67 @@ import {
   P1_VALUES,
 } from "./common";
 
+const PolkadotCLA = 0x90;
+const PolkadotSlip0044 = 0x80000162;
+
 interface TxMetadata {
   txMetadata: string;
 }
 
-export function newGenericApp(transport: Transport, chainTicker: string, txMetadataSrvUrl: string): GenericApp {
-  return GenericApp.new(transport, chainTicker, txMetadataSrvUrl);
+export function newGenericApp(transport: Transport, chainId: string, txMetadataSrvUrl: string): GenericApp {
```

Comment:
I'd export `newPolkadotApp` instead of `newGenericApp`. The idea is to unify everything under Polkadot and the app is becoming generic. Not considering this as a brand new app.

We have this that might cause some issues but we can make a breaking change and bump the version for this.
https://github.com/Zondax/ledger-substrate-js/blob/61cc9629bb735b2b9790e5b0125ec86984c5a72f/src/legacy_apps.ts#L25C1-L27

Reply:
It does not need to be a breaking change. If we do so, it won't be automatically taken by package dependency updates. I think it is better if we keep this backward compatible by choosing a different fn name!

Reply:
Renamed to newPolkadotGenericApp... let's do the breaking change after some time after we release this, and maybe polkadot community has migrated to the new polkadot app